README.md

```diff
@@ -22,8 +22,10 @@ Usage of ./bin/sql-gen-doc:
       a data source name for the database, e.g. user:password@tcp(mysql:3306)/database_name
   -o string
       the outfile to write the documentation to, if no outfile is specified, the output is written to stdout
+  --sort-tables
+      outputs tables in alphabetical order
```

Comment:
we should make this the default once we hit 1.0

otherwise I think the behavior is somewhat nondeterministic? (or based upon the order that the dbs are created)